test/common/setup-chain/actions/cache-connection.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+// To faciliate parallel testing, we cannot always share a connection or monkeypatching
+// one test can bleed into another test
+
+const Redis = require('ioredis')
+const log = require('../../log.js').child({module: 'keydb-connection'})
```

Comment:
would remove reference to keydb. or just always reference `redis`

Reply:
Good catch. C&p